### DISCUSSION TITLE
specify file containing usernames for downloading

### DISCUSSION THREAD

**Discussion by matt1544** [Serene-Arc/bulk-downloader-for-reddit#525] [Ideas]
Would it possible to specify a file which contains usernames for download?

I know that I can specify multiple "--user XYZ" within one command, but it may be preferable to have the usernames in a separate text file, and instead run "--user ~/Reddit/users.txt". 

I can probably figure out some _for_ loop to do this, but wondered if it'd be possible from within bdfr instead?

Thanks!  

(PS I suppose this request could also apply to "--subreddit" but that isn't the use case I was thinking of at the moment.)

**Comment by Serene-Arc** [comment]
There isn't a specific option for doing it but it's pretty easy to do through the command line. 

```bash
cat names.txt | xargs -L 1 echo '--user' | xargs -L X python3 -m bdfr ...
```

Just replace X with a number that works for you. If you have a reasonable number of names, you can remove the `-L X`, otherwise your shell might error out since there's a character limit for commands.
